kevm-pyk/src/kevm_pyk/kproj/evm-semantics/lemmas/bitwise-simplification.k

```diff
@@ -1,6 +1,7 @@
 requires "word.md"
 
 module BITWISE-SIMPLIFICATION [symbolic]
+    imports EVM-TYPES
```

Comment:
@ehildenb Any thoughts?

Reply:
Yes, or at least a specific module. But a specific file may be good too.

Reply:
Actually, I've just noticed, the `BITWISE-SIMPLIFICATION` module is already using the`WORD` module, so it is already EVM-specific. If we choose to separate those as well, I would suggest that we do this in a new PR after this one is merged. I think the lemmas overall could benefit from an overhaul, since there are a few specific ones.